test/intl402/Temporal/TimeZone/from/timezone-case-insensitive.js

```diff
@@ -3,10 +3,615 @@
 
 /*---
 esid: sec-temporal.timezone.from
-description: Time zone names are case insensitive
-features: [Temporal]
+description: Time zone identifiers are case-normalized
+features: [Temporal, time-zone-canonicalization]
 ---*/
 
-const timeZone = 'eTc/gMt+1';
-const result = Temporal.TimeZone.from(timeZone);
-assert.sameValue(result.id, 'Etc/GMT+1', `Time zone created from string "${timeZone}"`);
+const timeZoneIdentifiers = [
```

Comment:
This list should probably be alphabetical so you know where to look if you need to check whether a particular one is included.

Reply:
OK, will alphabetize.

Reply:
I realized that the list is already sorted, but it's separated into two parts: Zone names from TZDB and Link names from TZDB. That seems like a useful separation in case tests fail for only Links or only Zones. So I decided to leave the two groups as-is, but I added comments in front of each section to clarify what are Zones and what are Links.